index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Porfolio</title>
+    <title>Hans-Derly's Porfolio of work</title>
```

Comment:
- [optional] nice work.Dont you think `Hans-Derly's portfolio` will be a better title. please look into that

Reply:
I make the change, Thank you @tufoinnkuo10 !
Have a nice day